notebooks/notebook_template.ipynb

```diff
@@ -31,6 +31,8 @@
       "source": [
```

Comment:
<p>Does the authentication guide exist? The link doesn't work.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1204/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
No it will come in a separate PR.